pyta_server/routes.py

```diff
@@ -3,30 +3,39 @@
 import psycopg2
 from flask import request
 from pyta_server import app, db
-from pyta_server.models import Uploads
+from pyta_server.models import Submissions, Uploads
 
 @app.route('/', methods=['POST'])
 def receive():
     rand_hex = secrets.token_hex(8)
-    src_f = request.files.get('source')
+    src_f = request.files
     cfg_f = request.files.get('config')
-    id = request.form.get('id')
+    ident = request.form.get('id')
+    f_paths = []
     if src_f is not None and cfg_f is not None:
-        _, src_ext = os.path.splitext(src_f.filename)
+        for f in list(src_f.values())[:-1]: # Last value is cfg, handled separately
```

Comment:
Don't assume that the config file is the last one. Instead, loop through the keys inside this loop.